code/modules/mob/living/simple_animal/revenant/revenant_abilities.dm

```diff
@@ -139,26 +139,26 @@
 				if(T.flags & NOJAUNT)
 					T.flags -= NOJAUNT
 					PoolOrNew(/obj/effect/overlay/temp/revenant, T)
-				if(!istype(T, /turf/simulated/floor/plating) && !istype(T, /turf/simulated/floor/engine/cult) && istype(T, /turf/simulated/floor) && prob(15))
-					var/turf/simulated/floor/floor = T
-					if(floor.intact)
-						floor.builtin_tile.loc = floor
-					floor.broken = 0
-					floor.burnt = 0
-					floor.make_plating(1)
-				if(!istype(T, /turf/simulated/wall/shuttle) && !istype(T, /turf/simulated/wall/cult) && !istype(T, /turf/simulated/wall/rust) && !istype(T, /turf/simulated/wall/r_wall) && istype(T, /turf/simulated/wall) && prob(15))
-					PoolOrNew(/obj/effect/overlay/temp/revenant, T)
+					if(!istype(T, /turf/simulated/floor/plating) && !istype(T, /turf/simulated/floor/engine/cult) && istype(T, /turf/simulated/floor) && prob(15))
+						var/turf/simulated/floor/floor = T
+						if(floor.intact)
+							floor.builtin_tile.loc = floor
+						floor.broken = 0
+						floor.burnt = 0
+						floor.make_plating(1)
+					if(!istype(T, /turf/simulated/wall/shuttle) && !istype(T, /turf/simulated/wall/cult) && !istype(T, /turf/simulated/wall/rust) && !istype(T, /turf/simulated/wall/r_wall) && istype(T, /turf/simulated/wall) && prob(15))
+						PoolOrNew(/obj/effect/overlay/temp/revenant, T)
 					T.ChangeTurf(/turf/simulated/wall/rust)
 				if(!istype(T, /turf/simulated/wall/r_wall/rust) && istype(T, /turf/simulated/wall/r_wall) && prob(15))
 					PoolOrNew(/obj/effect/overlay/temp/revenant, T)
 					T.ChangeTurf(/turf/simulated/wall/r_wall/rust)
-				for(var/obj/structure/closet/closet in T.contents)
-					closet.open()
-				for(var/obj/structure/bodycontainer/corpseholder in T.contents)
-					if(corpseholder.connected.loc == corpseholder)
-						corpseholder.open()
-				for(var/obj/machinery/dna_scannernew/dna in T.contents)
-					dna.open_machine()
+					for(var/obj/structure/closet/closet in T.contents)
+						closet.open()
+					for(var/obj/structure/bodycontainer/corpseholder in T.contents)
+						if(corpseholder.connected.loc == corpseholder)
+							corpseholder.open()
+					for(var/obj/machinery/dna_scannernew/dna in T.contents)
+						dna.open_machine()
```

Comment:
Why indent any of these(lines 142 to 150 and 155 to 161)? Indenting them causes problems and makes the ability work improperly.